lib/go-atscfg/remapdotconfig.go

```diff
@@ -401,14 +470,16 @@ func buildEdgeRemapLine(
 	remapConfigParams []tc.Parameter,
 	cacheGroups map[tc.CacheGroupName]tc.CacheGroupNullable,
 	nameTopologies map[TopologyName]tc.Topology,
-) (string, []string, error) {
+) (RemapLines, []string, error) {
 	warnings := []string{}
+	var remapLines RemapLines
```

Comment:
Nitpick: the cache config code generally tries to use the idiom of always declaring variables with `:=`. 
There's no reason for Go to have multiple ways to declare variables, and IMO it's a slight readability improvement to always use `:=`